code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm

```diff
@@ -106,13 +106,31 @@
 	metabolization_rate = 0.05
 	taste_description = "salt"
 
+/datum/reagent/blackpowder/on_mob_life(mob/living/M)
+	M.losebreath += 10
+	if(holder.has_reagent("blackpowder"))
+		holder.remove_reagent("blackpowder", 10)
+		holder.add_reagent("dampblackpowder", 5)
```

Comment:
OH GOD WHY
fucking hell magic math

make it a blood catalyzed reaction or something jesus